client/src/deploy/creation_common.rs

```diff
@@ -777,6 +866,24 @@ pub(super) fn apply_common_session_options<'a, 'b>(subcommand: App<'a, 'b>) -> A
                 .arg(args_complex::session::ARG_NAME)
                 .required(false),
         )
+        .arg(session_package_hash::arg())
+        .arg(session_package_name::arg())
+        .arg(session_hash::arg())
+        .arg(session_name::arg())
+        .arg(session_entry_point::arg())
+        .arg(session_version::arg())
+}
+
+pub(crate) fn apply_common_purchase_options(
```

Comment:
where is this term `purchase` coming from?

Reply:
yeah that should be payment